README.md

```diff
@@ -1,42 +1,31 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+Mobile App VERSION
+Project Name
+MOBILE-VERSION
 
-# Project Name
+ Project Description.
```

Comment:
- [ ] as the previous reviewer asked, it would be better if you add a description about your projects for example `This is my portfolio website where I show my recent projects and my contact information built with HTML and CSS ` so please try to do that

Reply:
Implemented 👍